src/server/p2p/network.rs

```diff
@@ -786,9 +786,7 @@ where S: ShareChain
                 return;
             }
             let peer_store_read_lock = self.network_peer_store.read().await;
-            let my_best_peers = peer_store_read_lock.best_peers_to_share(NUM_PEERS_TO_SYNC_PER_ALGO, &self.squad, &[]);
-
-            let my_best_peers: Vec<_> = my_best_peers.into_iter().map(|p| p.peer_info).collect();
+            let my_best_peers = vec![];
```

Comment:
this will never return any best peers?

Reply:
The person who requests does not include any peers. The responder returns peers.